src/main.rs

```diff
@@ -3,17 +3,34 @@ extern crate env_logger;
 #[macro_use]
 extern crate failure;
 extern crate human_panic;
+extern crate log;
 extern crate structopt;
 extern crate wasm_pack;
 extern crate which;
 
 use std::env;
 use std::panic;
+use std::sync::mpsc;
+use std::thread;
 use structopt::StructOpt;
+use wasm_pack::command::build::Build;
 use wasm_pack::{command::run_wasm_pack, Cli};
 
 mod installer;
 
+fn background_check_for_updates() -> mpsc::Receiver<(String, String)> {
+    let (sender, receiver) = mpsc::channel();
+    let _detached_thread = thread::spawn(move || {
+        if let Ok((local, latest)) = Build::return_wasm_pack_versions() {
+            if !local.is_empty() && !latest.is_empty() && local != latest {
+                sender.send((local, latest)).unwrap();
```

Comment:
Do we really want to `unwrap()` here? cc @fitzgen @ashleygwilliams

Reply:
It doesn't really matter since we don't rely on the background thread to complete successfully.

It may be slightly tidier to do 

```rust
let _ = sender.send((local, latest));
```

to indicate that this is a best-effort attempt to notify the main thread, but it might have won the race to finish the command before the background check finished. But I think the point is fairly academic, given that if the main command finishes, then the main thread is shutting down and the whole process along with it, meaning that this background thread is going to get killed either way.